src/main/scala/ai/starlake/job/Main.scala

```diff
@@ -318,6 +318,9 @@ class Main() extends StrictLogging {
         new Yml2Xls(schemaHandler).run(args.drop(1))
         true
 
+      case "xls2ymljob" =>
+        Xls2YmlAutoJob.run(args.drop(1))
```

Comment:
Should’nt we just add the —job In the xml2yml command line ?

Reply:
simplified as requested, --job is a boolean opiton to choose between generating Domains OR Jobs